tests/kernel/queue/src/test_queue_loop.c

```diff
@@ -10,13 +10,13 @@
 #define LIST_LEN 4
 #define LOOPS 32
 
-static qdata_t data[LIST_LEN];
-static qdata_t data_p[LIST_LEN];
-static qdata_t data_r[LIST_LEN];
-static struct k_queue queue;
+APP_BSS_MEM static qdata_t data[LIST_LEN];
```

Comment:
None of these changes should be necessary, and we should not be putting kernel objects in application memory anyway